save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/storage/AbstractSimpleStorageUsingProjectReactor.kt

```diff
@@ -0,0 +1,34 @@
+package com.saveourtool.save.storage
+
+import com.saveourtool.save.s3.S3Operations
+import com.saveourtool.save.storage.key.AbstractS3KeyManager
+import com.saveourtool.save.storage.key.S3KeyManager
+
+/**
+ * S3 implementation of Storage
+ *
+ * @param s3Operations [S3Operations] to operate with S3
+ * @param prefix a common prefix for all S3 keys in this storage
+ * @param K type of key
+ */
+abstract class AbstractSimpleStorageUsingProjectReactor<K : Any>(
```

Comment:
Maybe improve naming? Like `AbstractSimpleReactiveStorage`?

Reply:
renamed `StorageUsingProjectReactor` to `ReactiveStorage`, but leave as is `StorageProjectReactor`